frontend/vite.config.ts

```diff
@@ -1,6 +1,7 @@
 import reactPlugin from '@vitejs/plugin-react';
 import { type ConfigEnv, defineConfig, loadEnv } from 'vite';
 import { VitePWA as vitePWAPlugin } from 'vite-plugin-pwa';
+import svgr from 'vite-plugin-svgr';
```

Comment:
let's rename as another plugins

Reply:
I assume it's [this one](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/75/commits/9ccea8dcdc0ab8060e3ab472e0f63f9843f03758)